modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/resource_selector/page.jsp

```diff
@@ -0,0 +1,47 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/resource_selector/init.jsp" %>
+
+<%
+String inputLabel = (String)request.getAttribute("liferay-frontend:resource-selector:inputLabel");
+String inputName = (String)request.getAttribute("liferay-frontend:resource-selector:inputName");
+String modalTitle = (String)request.getAttribute("liferay-frontend:resource-selector:modalTitle");
+String resourceName = (String)request.getAttribute("liferay-frontend:resource-selector:resourceName");
+long resourceValue = (long)request.getAttribute("liferay-frontend:resource-selector:resourceValue");
+String selectEventName = (String)request.getAttribute("liferay-frontend:resource-selector:selectEventName");
+String selectResourceURL = (String)request.getAttribute("liferay-frontend:resource-selector:selectResourceURL");
+boolean showRemoveButton = (boolean)request.getAttribute("liferay-frontend:resource-selector:showRemoveButton");
+String warningMessage = (String)request.getAttribute("liferay-frontend:resource-selector:warningMessage");
+%>
+
+<div>
+	<react:component
```

Comment:
It's a good practice to add some skeleton markup here, so the HTML does not appear empty until the JS has loaded.

I recommend this specially because it is a taglib that is going to be used in many places.